examples/crypto/SP800-108_counter_KDF/main.c

```diff
@@ -0,0 +1,400 @@
+/*
+ * TODO: Change to the right licence
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Please use an SPDX file copyright tag, either to replace or in addition to this comment. E.g.

```suggestion
 * SPDX-FileCopyrightText: Copyright (c) 2023 Nordic Semiconductor ASA
```